### ISSUE TITLE
Stair not oriented correctly

### ISSUE THREAD

**Issue by Scalion** [aleksilassila/litematica-printer#57]
Any stairs are not oriented properly and only depends on player position (shouldn't).
It should print only if you are in the good oriented position.

Also printing only above is super annoying at times, is there a way to bypass this?

**Comment by aleksilassila** [comment]
Is this happening on a specific server that has anticheat enabled  perhaps? Do you have the latest version of printer? Does lowering your printing speed help?

**Comment by Scalion** [comment]
Solo
Never mind, I really thought I had the latest version but after more digging and avoiding other external links I found your latest, thank you, work great!
The only thing I've noticed is sometimes I have some random block (of my build) that appears out of nowhere (as air block replacement, like anywhere, middle of the road let say).
And another similar bug, I have blocks of air (intentional) and sometimes it put a block over it. Once I delete that extra its fine. he won't do it again.
Which is why I go layer by layer while building to check if there's anything wrong.
